pyproject.toml

```diff
@@ -4,7 +4,6 @@ version = "0.1.0"
 description = ""
 authors = ["Aaron Haefner <aaronhaefner@uchicago.edu>"]
 readme = "README.md"
-packages = [{include = "new_arrivals_chi"}]
```

Comment:
The problem stemmed from two things:
1. You had this section here, which I don't believe you need. You are not gonna export this as a package at some point, as it's an application.
2. You lacked a base python package in your repository, so `pytest` didn't know what to reference as your non-test files. That was corrected by creating the `new_arrivals_chi` package that you can use as the base folder for your application.